tensorflow/lite/micro/BUILD

```diff
@@ -477,6 +477,17 @@ cc_test(
     ],
 )
 
+py_library(
+    name = "generate_cc_arrays",
+    srcs = ["tools/make/generate_cc_arrays.py"],
+)
+
+py_binary(
+    name = "bzl_generate_cc_arrays",
+    srcs = ["tools/make/bzl_generate_cc_arrays.py"],
+    deps = [":generate_cc_arrays"],
+)
```

Comment:
should we put the BUILD file in tools/make? That's a more common convention.

Or possibly a new folder (tools/data_generation)? I'm wondering if we can have a directory that can serve as a place for additional tools as well (similar to the lite/tools/ directory in the TF repo

Reply:
Moved build file and scripts to tensorflow/lite/micro/tools.